audio.go

```diff
@@ -42,6 +50,19 @@ type AudioRequest struct {
 	Temperature float32
 	Language    string // For translation, just do not use it. It seems "en" works, not confirmed...
 	Format      AudioResponseFormat
+
+	// TimeStampGranularities The timestamp granularities to populate for this transcription.
+	// response_format must be set verbose_json to use timestamp granularities.
+	// Either or both of these options are supported: word, or segment.
+	// Note: There is no additional latency for segment timestamps,
+	// but generating word timestamps incurs additional latency.
+	TimeStampGranularities []TimestampGranularity
+}
+
+type Word struct {
```

Comment:
nit: the name `Word` is very generic, could we please either change it to something like `AudioResponseWord` or make it anonymous similar to `Segments []struct {` in `AudioResponse`?